tests/config/test_data.config

```diff
@@ -549,5 +549,10 @@ params {
                 pretext = "${test_data_dir}/genomics/eukaryotes/galaxea_fascicularis/hic/jaGalFasc40_2.pretext"
             }
         }
+        'yahs' {
+            ilEupCent1_sorted_bed = "https://darwin.cog.sanger.ac.uk/sanger-tol-genomeassembly/test/yahs/ilEupCent1.sorted.bed" 
```

Comment:
You should use existing data from the [nf-core test data repository](https://nf-co.re/docs/contributing/test_data_guidelines), or else PR some new (small) test files if there's nothing suitable.

These will be used in CI etc, and we don't want to be hammering the Sanger servers repeatedly.